e2e/appActions.js

```diff
@@ -656,6 +656,26 @@ async function renameObjectFromContextMenu(page, url, newName) {
   await page.click('[aria-label="Save"]');
 }
 
+/**
+ * @param {import('@playwright/test').Page} page
+ */
+async function turnOffAutoscale(page) {
+  // uncheck autoscale
+  await page.getByRole('checkbox', { name: 'Auto scale' }).uncheck();
+}
+
```

Comment:
Maybe we have a `plotActions` or something? The other issue in generalizing this function is that it will apply to all plots on a screen, so either the function name should probably reflect that (which is fine for now), or the function should have a parameter that specifies a particular plot (which is more work). I'd be fine with the former.

Reply:
yeah this feels like it might be better as a one-liner and a comment in the test itself rather than abstracting

Reply:
It was used in 2 tests so I will move it into a plotActions as suggested by @akhenry